### ISSUE TITLE
non-resident keys maybe?

### ISSUE THREAD

**Issue by My1** [rgerganov/solo-pgp#2]
it would in my opinion also be kinda nice to be able to use normal keys instead of only RKs for this

**Comment by rgerganov** [comment]
Yeah, I thought about this too. The benefit of using RK is that you don't need to keep any state on the host. With normal credentials we have to keep the mapping between PGP key ID -> credential_id on the host.
